controllers/constants/constants.go

```diff
@@ -36,8 +36,8 @@ const (
 	QservName = "qserv"
 	DotQserv  = "dot-qserv"
 
-	ReplicationControllerPort     = 8080
-	ReplicationControllerPortName = "http"
+	HTTPPort     = 8080
```

Comment:
It looks like the Master Controller and the Registry are going to listen to the same port. I guess this means that these services would have to be run in different pods. Is this your plan?